dev-python/sphinx-press-theme/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>paul.zander+gentoo@gmail.com</email>
```

Comment:
Have you registered to our bugzilla with this mail?

Reply:
Fixed.